css/intro.css

```diff
@@ -86,8 +87,8 @@
 }
 
 @media screen and (max-width: 500px) {
-  .input-label-dark {
-    font-size: 0.9rem;
+  .input-big {
+    height: 42rem;
```

Comment:
are you sure this is not too big of a value?
(42rem = 42 x 16px ==> 672px)

Reply:
@yesyash as superuser don't want to scroll the information input field

https://user-images.githubusercontent.com/107163260/231949128-f4d7780d-298d-4565-ac60-f29f8fd8562b.mp4